fieldservice_account_analytic/data/ir_rule.xml

```diff
@@ -0,0 +1,24 @@
+<odoo>
+    <record id="analytic_account_fsm_manager" model="ir.rule">
+        <field name="name">account.analytic.line.fsm.manager</field>
+        <field name="model_id" ref="model_account_analytic_line" />
+        <field name="global" eval="False" />
+        <field name="domain_force">[(1, '=', 1)]</field>
```

Comment:
This is coming from previous versions, but it's not correct.
It grants too much power.

It should only grant access to lines having a `fsm_order_id`, IMO:

```suggestion
        <field name="domain_force">[('fsm_order_id', '!=', False)]</field>
```